### PR TITLE
Adds 2023 PIF cohorts to site

### LINKED ISSUE #542: PIF FY23 Cohort Page
Hello! Here's a link to the spreadsheet for the new cohort page. This year, the team decided not to include project descriptions (so yeah, less work for you and more consistency on the PIF website! :)). Please let me know if you have any questions as you build the page. https://docs.google.com/spreadsheets/d/1cl56Ius9DPY2g5HDb3HOMXVgYHIjBJ1H2s0Y6YyXH2o/edit#gid=0

### PR DESCRIPTION
**Pull request by jaredbain**
Fixes #542

### CODE REVIEW

File: _fellows/beatrice-mercier.md
Comment (Marcia-Simms) [inline _fellows/beatrice-mercier.md:1]: Beatrice's name has an accent over the first "e"

File: _fellows/rajni-aneja.md
Comment (Marcia-Simms) [inline _fellows/rajni-aneja.md:1]: "health/digital health and data"

File: _fellows/rupam-jyoti-sarmah.md
Comment (Marcia-Simms) [inline _fellows/rupam-jyoti-sarmah.md:1]: Change "twenty-five" to "25"

Comment (Marcia-Simms) [reply _fellows/rupam-jyoti-sarmah.md:1]: Change "thirty" to "30"

File: _fellows/patriek-karayil.md
[the remark this answers is not in the document]
Comment (jaredbain) [reply _fellows/patriek-karayil.md:1]: A bio was not provided. It's empty on the google sheet.

File: _fellows/sailaja-savanam.md
Comment (Marcia-Simms) [inline _fellows/sailaja-savanam.md:1]: "She is a technology leader with a data-driven approach."

File: _fellows/yll-agimi.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ agency: ["DOD"]
...
-tags: ['2023']
-skills: []
+tags: ['2023', 'east', 'Strategy_And_Growth']
+skills: ['Strategy and Growth']
 ---
 
 Yll Agimi (he/him) is a Data Scientist covering the domains of machine leaning, statistics and healthcare. Most recently, Yll worked at General Dynamics Information Technology as a Senior Data Scientist, leading a team building advanced data solutions for clients, including the Military Health System (MHS), the health system supporting some 10 million active-duty personnel, military retirees and their families. Yll partnered with senior stakeholders to scope, develop and deliver new innovative approaches to incorporating data science into aspects of healthcare operations and clinical decision making. He oversaw the development and use of algorithms and statistical predictive models to evaluate health outcomes. He led teams in determining analytical approaches and modeling techniques. Supported clients in extracting insights out of structured and unstructured data to solve complex business problems by relying on statistical techniques and broad knowledge of healthcare systems.
```
Comment (Marcia-Simms) [inline _fellows/yll-agimi.md:15]: "machine leaning" should be "machine learning"

File: _fellows/yll-agimi.md
Comment (Marcia-Simms) [inline _fellows/yll-agimi.md:1]: While at Booz Allen Hamilton, Yll et al. built the “Chronic Disease and Healthcare Cost Predictor” a personal disability predictor tool: a tool to model the probability of disease using user data and data from the Behavioral Risk Factor Surveillance System.

**Review by Marcia-Simms — COMMENTED** [review]
Had a few edits

File: _fellows/beatrice-mercier.md
Comment (jaredbain) [reply _fellows/beatrice-mercier.md:1]: Is it a grave or acute accent?

File: _fellows/patriek-karayil.md
[the remark this answers is not in the document]
Comment (Marcia-Simms) [reply _fellows/patriek-karayil.md:1]: Patriek is a C-level executive with strong experience in Digital Transformation. He has been a career CMO with an established track record of taking large enterprises through innovative growth phases - from immersive experiential models, to better decision making, to game changing efficiencies, to modernizing operations, and to redefining customer-centricity; all on the backs of disruptive digital technologies. 

He holds a Bachelors in Computer Science from Arizona State University and an MBA from Massachusetts Institute of Technology (MIT).

**Comment by jaredbain** [comment]
Applied all the edits to the staging link.

File: _fellows/pamela-jennings-phd.md [hunk 2]
```diff
@@ -12,4 +12,8 @@ tags: ['east','2022',Change_Management,Strategy_And_Growth]
...
...edicine in Higher Education: Branches from the Same Tree”. Pamela has developed program frameworks that integrate learning, research, creativity, innovation, and economic development. She believes it is the nexus of these platforms where breakthroughs and paradigm shifts are made. In her career she has experienced, the juxtaposition of open-innovation research and development in corporate think tanks; the possibilities of generating creatively applied research at federally funded think tanks; the differences in learning models from the arts to the sciences; how the arts and humanities can generate speculative inquiry that informs STEM research and vice versa; the constrictions of the possible when institutional parochialism is too strong; the creative genius that blossoms when given, space and time in places that embrace and nurture the imagination; the power budgets give to influence the course of critical inquiry, policy, innovation, and change; the ability to greatly increase research and development output and impact with small resource shifts; the complexity of building a space for integrative research and creativity that balances government mandates with grassroot enthusiasm.
```
Comment (Marcia-Simms) [inline _fellows/pamela-jennings-phd.md:17]: "human-centered"
"Human-Centered"
"National Science Foundation (NSF)"
Tree."
"In her career, she has experienced: 

- the juxtaposition of open-innovation research and development in corporate think tanks. 
- the possibilities of generating creatively applied research at federally funded think tanks. 
- the differences in learning models from the arts to the sciences; how the arts and humanities can generate speculative inquiry that informs STEM research and vice versa. 
- the constrictions of the possible when institutional parochialism is too strong. 
- the creative genius that blossoms when given, space and time in places that embrace and nurture the imagination.
- the power budgets give to influence the course of critical inquiry, policy, innovation, and change.
- the ability to greatly increase research and development output and impact with small resource shifts. 
- the complexity of building a space for integrative research and creativity that balances government mandates with grassroot enthusiasm."

File: _fellows/rupam-jyoti-sarmah.md
Comment (Marcia-Simms) [inline _fellows/rupam-jyoti-sarmah.md:1]: "Ph.D."

File: _fellows/yll-agimi.md
Comment (Marcia-Simms) [inline _fellows/yll-agimi.md:1]: "Ph.D."

File: _fellows/adam-roberts.md
Comment (Marcia-Simms) [inline _fellows/adam-roberts.md:1]: Just flagging again ...still internal discussions happening

**Review by Marcia-Simms — APPROVED** [review]

**Pull request merged**